examples/check.py

```diff
@@ -34,16 +34,7 @@
     description = 'Example bank account',
     routing_number = '122100024',
     account_number = '1234564789',
-    signatory = 'John Doe',
-    bank_address = {
-        'name': 'Bank Address',
-        'address_line1': '123 Wall Street',
-        'address_city': 'San Francisco',
-        'address_state': 'CA',
-        'address_zip': '94158',
-        'address_country': 'US'
-    },
-    account_address = example_address
```

Comment:
good catch, I ended up reusing it as the `check.to_address` since the `data` param was using `example_address.name` for dynamic html.